index.coffee

```diff
@@ -82,91 +82,89 @@ class ProgressReporter
 
 	# Create a stream that transforms `docker.modem.followProgress` onProgress
 	# events to include total progress metrics.
-	pullProgress: (image, onProgress) ->
+	pullProgress: Promise.method (image, onProgress) =>
```

Comment:
The `=>` isn't needed as `Promise.method` will call the function with the same `this`, and essentially be equivalent to `-> fn.apply(this, arguments)` but with the promise wrapping